content/en/real_user_monitoring/frustration_signals/_index.md

```diff
@@ -0,0 +1,155 @@
+---
+title: Frustration Signals
+kind: documentation
+further_reading:
+- link: '/real_user_monitoring/dashboards/frustration_signals_dashboard/'
+  tag: 'Documentation'
+  text: 'Frustration Signals Dashboard'
+- link: '/real_user_monitoring/explorer'
+  tag: 'Documentation'
+  text: 'Learn about the RUM Explorer'
+- link: '/real_user_monitoring/session_replay'
+  tag: 'Documentation'
+  text: 'Learn about Session Replay'
+---
+
+## Overview
+
+<div class="alert alert-warning">
+For access to frustration signals, contact <a href="/help">Datadog Support</a>.
+</div>
+
+Frustration signals identify the highest points of user friction in your application by surfacing moments where users exhibit frustration. 
+
+RUM collects three types of frustration signals:
+
+Rage Clicks
+: A user clicks on an element more than three times in a one-second rolling window.
```

Comment:
```suggestion
: A user clicks on an element more than three times in a one-second sliding window.
```

they mean the same thing, but the other one said "sliding" so let's stick with sliding